### PR TITLE
feat: add trivy job

### LINKED ISSUE #185: feat: add trivy job
https://github.com/aquasecurity/trivy#gitlab-ci

### PR DESCRIPTION
**Pull request by revolunet**
fix #185

### CODE REVIEW

**Comment by revolunet** [comment]
up :)

**Comment by revolunet** [comment]
ding dong

File: base_trivy_scan.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  allow_failure: true
+  script:
+    # Build image
+    - docker build -t $IMAGE .
+    # Build report
+    - ./trivy -d --exit-code 0 --cache-dir .trivycache/ --no-progress --format template --template "@contrib/gitlab.tpl" -o gl-container-scanning-report.json $IMAGE
+    # Print report
+    - ./trivy -d --exit-code 0 --cache-dir .trivycache/ --no-progress $IMAGE
+    # Fail on high and critical vulnerabilities
+    - ./trivy -d --exit-code 1 --cache-dir .trivycache/ --no-progress $IMAGE
```
Comment (douglasduteil) [inline base_trivy_scan.yml:29]: We want a socialgouv/docker/trivy image :)

Comment (revolunet) [reply base_trivy_scan.yml:29]: theres nothing really specific ?

Comment (douglasduteil) [reply base_trivy_scan.yml:29]: @renovate-bot :eye: https://github.com/SocialGouv/docker/pull/228

**Pull request merged**